spec.html

```diff
@@ -472,16 +472,12 @@ <h1>ShadowRealm ( )</h1>
 			<emu-alg>
 				1. If NewTarget is *undefined*, throw a *TypeError* exception.
 				1. Let _O_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%ShadowRealm.prototype%"*, « [[ShadowRealm]], [[ExecutionContext]] »).
-				1. Let _realmRec_ be CreateRealm().
-				1. Set _O_.[[ShadowRealm]] to _realmRec_.
-				1. Let _context_ be a new execution context.
-				1. Set the Function of _context_ to *null*.
-				1. Set the Realm of _context_ to _realmRec_.
-				1. Set the ScriptOrModule of _context_ to *null*.
+				1. Perform ? InitializeHostDefinedRealm().
+				1. Let _context_ be the running execution context.
 				1. Set _O_.[[ExecutionContext]] to _context_.
-				1. Perform SetRealmGlobalObject(_realmRec_, *undefined*, *undefined*).
-				1. Perform ? SetDefaultGlobalBindings(_O_.[[ShadowRealm]]).
-				1. Perform ? HostInitializeShadowRealm(_O_.[[ShadowRealm]]).
+				1. Let _realmRec_ be the Realm of _context_.
+				1. Set _O_.[[ShadowRealm]] to _realmRec_.
+				1. Perform ? HostInitializeShadowRealm(_realmRec_).
```

Comment:
What is the execution context after this? It seems to me that the new `InitializeHostDefinedRealm` "Push _newContext_ onto the execution context stack; _newContext_ is now the running execution context.", which is not the intent of `new ShadowRealm()`. Either `InitializeHostDefinedRealm` needs some work, or we must restore the execution context after calling it, isn't it?

Reply:
Good catch. I've changed this PR; I'm not planning to raise this with 262 myself, but can adjust here if someone else does.